data/db-schema/src/main/resources/net/corda/db/schema/crypto/migration/crypto-creation-v1.0.xml

```diff
@@ -22,9 +29,10 @@
         </createTable>
         <addPrimaryKey  columnNames="alias"
                         tableName="crypto_wrapping_key"
-                        constraintName="crypto_wrapping_key_pk"/>
+                        constraintName="crypto_wrapping_key_pk"
+                        schemaName="${schema.name}"/>
 
-        <createTable tableName="crypto_hsm_config">
+        <createTable tableName="crypto_hsm_config" schemaName="${schema.name}">
```

Comment:
Similar to the above, schemas were left out as they should be passed into the connection string to this DB.